pom.xml

```diff
@@ -780,7 +780,12 @@
          comes in via hbase-thirdparty hbase-shaded-netty-->
     <netty.hadoop.version>3.10.5.Final</netty.hadoop.version>
     <!-- end HBASE-15925 default hadoop compatibility values -->
-    <audience-annotations.version>0.5.0</audience-annotations.version>
+    <audience-annotations.version>0.13.0</audience-annotations.version>
+    <!--
+      the version used when generating javadoc
+      we need to use a newer version when building with jdk11
+    -->
+    <javadoc.audience-annotations.version>0.13.0</javadoc.audience-annotations.version>
```

Comment:
this should be 0.14.1, to use the RC repository.

Reply:
0.14.0 is for JDK11 only, it can not be used together with JDK8. See here

https://github.com/apache/yetus/releases

the audience-annotations section.

Reply:
Ah okay. Please add a comment to this effect, this is the last version of Yetus that we can use until we give up on JDK8. Thanks.

Reply:
OK, will update the PR once yetus 0.14.1 is out.